File: 2.0/java/calls/addCallBroadcastBatchPoliticalPoll.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.request.AddBatchRequest;
import com.callfire.api.client.api.common.model.ResourceId;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("api_login", "api_password");
        AddBatchRequest request = AddBatchRequest.create()
            .campaignId(11646003L)
            .name("Contacts Batch 1")
            // scrub duplicate numbers if any
            .scrubDuplicates(true)
            // add all contacts from previously created contact list
            .contactListId(300555001L)
            .build();
        ResourceId resourceId = callfireClient.callBroadcastsApi().addBatch(request);
    }
}

File: 2.0/csharp/calls/addCallBroadcastBatchPoliticalPoll.cs
using CallfireApiClient;
using CallfireApiClient.Api.Campaigns.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var request = new AddBatchRequest
        {
            // your existing campaign id, you can get it via find() operation
            CampaignId = 11646003,
            Name = "Contacts Batch 1",
            // id of contact list that you have added on previous step
            ContactListId = 300555001,
            ScrubDuplicates = true
        };
        var id = client.CallBroadcastsApi.AddBatch(request);
    }
}
